README.md

```diff
@@ -47,7 +47,7 @@ zgen load carloscuesta/materialshell materialshell
 Add the following snippet to your `.zshrc` file:
 
 ```sh
-zplug carloscuesta/materialshell, use:materialshell, from:github, as:theme
+zplug carloscuesta/materialshell, from:github, as:theme
```

Comment:
Hey!

Not sure if I follow but according to the documentation, if we don't specify `use` everything `*.zsh` will load why not specifying the only file that needs to be loaded `materialshell.zsh` 🤔 ?

https://github.com/carloscuesta/materialshell/blob/master/materialshell.zsh

Reply:
lol thats what i originally had in the initial commit. i'll revert